Source/Core/Core/GeckoCodeConfig.cpp

```diff
@@ -25,7 +25,7 @@ void LoadCodes(const IniFile& globalIni, const IniFile& localIni, std::vector<Ge
 
     for (auto& line : lines)
     {
-      if (line.empty())
+      if (line.empty() || line[0] == '#')
```

Comment:
you can just do:

```cpp
lines.erase(std::remove_if(lines.begin(), lines.end(), [](const auto& line) { return line.empty() || line[0] == '#' }), lines.end());
```

prior to entering the loop to just remove all of those types of lines from the vector before the parsing.